.github/workflows/check-heap-memory-violations.yml

```diff
@@ -0,0 +1,30 @@
+name: check-heap-memory-violations
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    if: ${{ startsWith(github.ref , 'refs/tags') != true && startsWith(github.ref , 'build-') != true }}
+    timeout-minutes: 90
+    name: Check Heap Memory Violations
+    steps:
+      - uses: actions/checkout@v2
+      - name: 'Print env'
+        run: |
+          echo "'uname -s' is:"
+          echo "uname: " $(uname)
+          echo "uname -m: " $(uname -m)
+          echo "uname -r:" $(uname -r)
+          echo "uname -s: " $(uname -s)
+          echo "uname -v: " $(uname -v)
+          printenv
+        shell: bash
+
+      - name: Check formatting (linux only)'
```

Comment:
bad 'name' for heap memory api violation checks?

Reply:
Fixed, thanks!